jgeorge37/acmwosu-db
# Szm/ada 85 front end meeting creation

To test in dev:
1. Sign into http://dev-ada.herokuapp.com/
2. Go to Exec Dashboard tab and click on "Meetings"
3. Fill in the information for adding a meeting and check whether the meeting was created in the table. If a company was associated with a meeting, check to make sure the meeting_company table has been updated correctly.
